src/catkin_pkg/changelog_generator_vcs.py

```diff
@@ -146,11 +146,11 @@ def _get_author(self, hash_):
         return result['output']
 
     def get_tags(self):
-        cmd_tag = [self._executable, 'tag']
+        cmd_tag = [self._executable, 'log', '--simplify-by-decoration', '--pretty="%d"']
         result_tag = self._run_command(cmd_tag)
         if result_tag['returncode']:
             raise RuntimeError('Could not fetch tags:\n%s' % result_tag['output'])
-        tag_names = result_tag['output'].splitlines()
+        tag_names = re.findall("(?<=tag: )([^)]+)", result_tag['output'])
```

Comment:
The regular expression uses a lookbehind which seems a bit too complicated to me. Can you please simply the regex to not require that.

The regex could also be made more specific by ensuring that the tag name:
* is preceeded by either `(` or `\s` and
* is followed by either `,` or `)`.

Please use single quotes instead of double quotes as in the rest of the file.